### DISCUSSION TITLE
FR: What about Progress Bar package?

### DISCUSSION THREAD

**Discussion by Klaymr** [charmbracelet/huh#553] [Ideas]
Like Spinner but with progress and animation, and can set by percent...

**Comment by bashbunni** [comment]
Hey @Klaymr, thank you so much for wanting to make this project better. I've converted this to a discussion so we can talk through it and keep our backlog clear. If it does end up being something we want to add, we'll promote it back to an issue when we are ready to work on it.

If we commit to this feature, that's a feature we will need to maintain for the lifespan of this project. So, since "Yes is forever", we are carefully considering new features.

As I mentioned in your other feature request, it would be super if you could provide us with a bit more information as to why you might want these components. Each new component/feature we add increases the maintenance load of the project, so it's really helpful for us to have as much context as possible :)
